graphql/schema.graphql

```diff
@@ -0,0 +1,5 @@
+type Query {
+  suggestAddresses(searchTerm: String): [AddressSuggestion]
+
+  getAddressByExternalId(id: String): Address
```

Comment:
Is there a way to import the `Address` type from the https://www.github.com/vtex-apps/places-graphql ?

Reply:
I dont' think so :-/